src/components/DatePicker/DatePicker.tsx

```diff
@@ -27,6 +28,14 @@ const DatesPicker: React.FC<DatesPickerProps> = ({formik, ...props}: DatesPicker
             formik.setFieldValue(props.fieldName, formattedDate);
         }
     }
+    const selectLabel = (type: string) => {
```

Comment:
I think it would be a good think to move this logic to another file, maybe a utils file, I would like to decouple this logic since the Date picker component could be use by another file.

Reply:
Jorge pointed out to me that there was a prop.label that could be used for that data, the logic was removed. Resolved :)